### ISSUE TITLE
[Bug]: lots of error "invalid database xxxx" in dn  when running sysbench test on distributed mode 

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#14384]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch Name

1.1-dev

### Commit ID

7a96a59

### Other Environment Information

```Markdown
- Hardware parameters:
3*CN: 16C 64G
1*DN: 16C 64G
3*LOG: 4C 16G
- OS type:
- Others:
```

### Actual Behavior

when running sysbench test on distributed mode, lots of error "invalid database sbtest" in cn ,but the queries were succeeded.

the error log:

{"level":"ERROR","time":"2024/01/25 11:01:27.274101 +0000","caller":"txnimpl/antidepend.go:42","msg":"error: r-w conflict"}
{"level":"ERROR","time":"2024/01/25 11:01:37.172192 +0000","caller":"txnimpl/antidepend.go:42","msg":"error: r-w conflict"}
{"level":"ERROR","time":"2024/01/25 11:01:57.244518 +0000","caller":"catalog/catalog.go:1110","msg":"error: invalid database sbtest"}
{"level":"ERROR","time":"2024/01/25 11:02:02.310749 +0000","caller":"catalog/catalog.go:1110","msg":"error: invalid database sbtest"}
{"level":"ERROR","time":"2024/01/25 11:02:12.191108 +0000","caller":"catalog/catalog.go:1110","msg":"error: invalid database sbtest"}
{"level":"ERROR","time":"2024/01/25 11:02:16.882988 +0000","caller":"txnimpl/antidepend.go:42","msg":"error: r-w conflict"}
{"level":"ERROR","time":"2024/01/25 11:02:22.290797 +0000","caller":"catalog/catalog.go:1110","msg":"error: invalid database sbtest"}
{"level":"ERROR","time":"2024/01/25 11:02:27.142719 +0000","caller":"catalog/catalog.go:1110","msg":"error: invalid database sbtest"}
{"level":"ERROR","time":"2024/01/25 11:02:32.030791 +0000","caller":"catalog/catalog.go:1110","msg":"error: invalid database sbtest"}
{"level":"ERROR","time":"2024/01/25 11:03:22.198804 +0000","caller":"catalog/catalog.go:1110","msg":"error: invalid database sbtest"}
{"level":"ERROR","time":"2024/01/25 11:03:46.409940 +0000","caller":"txnimpl/antidepend.go:42","msg":"error: r-w conflict"}
{"level":"ERROR","time":"2024/01/25 11:03:47.885174 +0000","caller":"catalog/catalog.go:1110","msg":"error: invalid database sbtest"}
{"level":"ERROR","time":"2024/01/25 11:03:58.874457 +0000","caller":"catalog/catalog.go:1110","msg":"error: invalid database sbtest"}

loki link:
http://192.168.240.6:20001/explore?panes=%7B%22ukw%22:%7B%22datasource%22:%22loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bnamespace%3D%5C%22mo-stability-regression-20240125%5C%22,%20pod%3D%5C%22stability-regression-dis-dn-0%5C%22%7D%20%7C%3D%20%60invalid%20database%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki%22%7D,%22editorMode%22:%22builder%22%7D%5D,%22range%22:%7B%22from%22:%221706158938476%22,%22to%22:%221706163460601%22%7D%7D%7D&schemaVersion=1&orgId=1

### Steps to Reproduce

```Markdown
1. run a distributed mo
2. runing sysbench mixed cases
```

**Comment by XuPeng-SH** [comment]
@gouhongshen

**Comment by gouhongshen** [comment]
blocking by s-1

**Comment by jiangxinmeng1** [comment]
还没进展

**Comment by XuPeng-SH** [comment]
fixed by #14622

**Comment by aressu1985** [comment]
fixed
